docs/kyma/04-02-local-deployment.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deploy Kyma locally
+type: Installation
+---
+
+This Deployment guide shows you how to quickly deploy Kyma locally on the MacOS, Linux, and Windows platforms. Kyma is deployed locally using a proprietary installer based on a [Kubernetes operator](https://coreos.com/operators/).
+
+**NOTE:** Local deployment using Minikube is deprecated.
+
+>**TIP:** See the [troubleshooting guide](#troubleshooting-overview) for tips.
+
+## Prerequisites
+
+<div tabs name="prerequisites" group="deploy-kyma-locally">
+  <details>
+  <summary label="k3s">
+  k3s
+  </summary>
+
+- [Kyma CLI](https://github.com/kyma-project/cli)
+- [Docker](https://www.docker.com/get-started)
+- [k3s](https://k3s.io/) 
+
+</details>
+  <details>
+  <summary label="minikube-deprecated">
```

Comment:
Same topic as below. If we want to document the deprecated way, we need to tell more about the difference. Maybe let's postpone the decision till we know if we will support the deprecated mode at all and how long